rpi-eeprom-update

```diff
@@ -627,6 +632,20 @@ checkAndApply()
       BOOTLOADER_UPDATE_IMAGE=""
    fi
 
+   # If the shell is not interactive then skip silent upgrades for the default/critical release
+   # if the bootloader is already new enough. This avoids silent upgrades unless
+   # the minimum version is advanced but preserves the behaviour of "rpi-eeprom-update -a"
+   if ! tty -s; then
+      if [ -n "${BOOTLOADER_AUTO_UPDATE_MIN_VER}" ] && [ "${ACTION_UPDATE_BOOTLOADER}" = 1 ]; then
+         if [ "${FIRMWARE_RELEASE_STATUS}" = "critical" ] || [ "${FIRMWARE_RELEASE_STATUS}" = "default" ]; then
+            if [ "${BOOTLOADER_CURRENT_VERSION}" -ge "${BOOTLOADER_AUTO_UPDATE_MIN_VER}" ]; then
+               echo "Skip silent bootloader upgrade current: ${BOOTLOADER_CURRENT_VERSION} >= min ${BOOTLOADER_AUTO_UPDATE_MIN_VER}"
```

Comment:
Super picky, but if there's a colon after `current` should there also be a colon after `min`?

Reply:
the colon should be removed